### ISSUE TITLE
[Bug] casaos-local-storage无限占用内存导致卡死

### ISSUE THREAD

**Issue by sesame-foxes** [IceWhaleTech/CasaOS#646]
![ING8@E(1F_QFEA 2ACG6X5H](https://user-images.githubusercontent.com/84228109/199172891-66055946-1206-4c8c-a689-59de46eefbdf.png)
casaos-local-storage 会随着使用无限增长最终导致系统卡死

**Comment by Mace404** [comment]
Got the same behaviour. casaos-local-storage service increasing memory over time.
Ran a while without any extra apps enabled but memory kept increasing.

from the log:

```
2022-10-31T21:19:51.949Z        info    Checking serial disk mount...   {"func": "service.(*diskService).CheckSerialDiskMount", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 469}
2022-10-31T21:19:51.996Z        error   failed to enable S.M.A.R.T: smartctl 7.2 2020-12-30 r5155 [x86_64-linux-5.18.17-edge] (local build)
Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

Smartctl open device: /dev/sda failed: No such device
        {"error": "exit status 2", "path": "/dev/sda", "func": "service.(*diskService).CheckSerialDiskMount", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 493}
2022-10-31T21:19:52.016Z        error   failed to enable S.M.A.R.T: smartctl 7.2 2020-12-30 r5155 [x86_64-linux-5.18.17-edge] (local build)
Copyright (C) 2002-20, Bruce Allen, Christian Franke, www.smartmontools.org

Smartctl open device: /dev/sdb failed: No such device
        {"error": "exit status 2", "path": "/dev/sdb", "func": "service.(*diskService).CheckSerialDiskMount", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 493}
2022-10-31T21:19:52.076Z        error   failed to exec shell - smartctl exec error      {"func": "service.(*diskService).SmartCTL", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 88}
2022-10-31T21:19:52.097Z        error   failed to exec shell - smartctl exec error      {"func": "service.(*diskService).SmartCTL", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 88}
2022-10-31T21:19:52.122Z        info    Notified systemd that local storage service is ready    {"func": "main.main", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/main.go", "line": 243}
2022-10-31T21:19:52.122Z        info    LocalStorage service is listening...    {"address": "127.0.0.1:35417", "func": "main.main", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/main.go", "line": 248}
2022-10-31T21:29:55.023Z        error   failed to exec shell - smartctl exec error      {"func": "service.(*diskService).SmartCTL", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 88}
2022-10-31T21:29:55.045Z        error   failed to exec shell - smartctl exec error      {"func": "service.(*diskService).SmartCTL", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 88}
2022-10-31T21:40:00.024Z        error   failed to exec shell - smartctl exec error      {"func": "service.(*diskService).SmartCTL", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 88}
2022-10-31T21:40:00.045Z        error   failed to exec shell - smartctl exec error      {"func": "service.(*diskService).SmartCTL", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 88}
2022-10-31T21:50:05.024Z        error   failed to exec shell - smartctl exec error      {"func": "service.(*diskService).SmartCTL", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 88}
2022-10-31T21:50:05.046Z        error   failed to exec shell - smartctl exec error      {"func": "service.(*diskService).SmartCTL", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 88}
2022-10-31T22:00:10.022Z        error   failed to exec shell - smartctl exec error      {"func": "service.(*diskService).SmartCTL", "file": "/home/runner/work/CasaOS-LocalStorage/CasaOS-LocalStorage/service/disk.go", "line": 88}
```

Where it repeats every 10 minutes.

restarting the service with  "systemctl restart casaos-local-storage.service" releases the memory

**Comment by Black-Boom** [comment]
Same problem using v0.3.7

**Comment by jerry945** [comment]
![image](https://user-images.githubusercontent.com/5111166/199632872-f3de60bc-64c5-4003-93ab-d4d31879da80.png)

**Comment by zhong1260310** [comment]
3个设备更新了 0.3.7版本    ， 同样问题，  0.3.6 没有这种问题

**Comment by raller1028** [comment]
This issue has been fixed in 0.3.7.1, please verify in the update to 0.3.7.1, if you have problems, please reopen the issue.
